Form/BaseType.php

```diff
@@ -49,21 +41,6 @@ protected function checkCredentials($credentials, $model)
     }
 
     /**
-     * This method is used to pass the securityContext into custom formTypes.
-     *
-     * @param string|FormTypeInterface $formType
-     * @return string|FormTypeInterface
-     */
-    protected function inject($formType)
-    {
-        if (is_object($formType) && method_exists($formType, 'setAuthorizationChecker')) {
-            $formType->setAuthorizationChecker($this->authorizationChecker);
-        }
-
-        return $formType;
-    }
-
```

Comment:
@sescandell i really like that auto-registering as service and removeing this part of code, it was kinda hacky